ticket_app/src/components/SprintInfo.jsx

```diff
@@ -1,80 +1,152 @@
-                    <h2 className="text-2xl underline">In Progress</h2>
-                    {inProgressTickets.map((ticket) => (
-                        <h3 onClick={() => {navigate(`${company.id}/ticket/${ticket.id}`)}} key={ticket.id}>{ticket.feature}</h3>
-                    ))}
-                </div>
-                <div className="text-2xl underline basis-1/4 text-center bg-indigo-400">Testing</div>
-                <div className="text-2xl underline basis-1/4 text-center bg-blue-300">Verified</div>
-                <div className="text-2xl underline basis-1/4 text-center bg-green-400">Approved</div>
-            </div>
+  return (
+    <div className=" lg w-full h-screen">
+      <div className="flex mt-4 bg-pink-100 h-20 w-full justify-around align-middle py-4">
+        <h2 className="my-auto text-4xl font-bold">Sprint: {sprint.name}</h2>
+        <div>
+          <h3>{`Start date: ${sprint.start_date}`}</h3>
+          <h3>{`End date: ${sprint.end_date}`}</h3>
         </div>
-    )
-}
+      </div>
 
-    
+      {/* Start of sprint information board */}
+      
+      <div className="mt-4 w-full h-full flex justify-around">
+        <div className="basis-1/4 text-center">
+          <TicketGroup
+            heading="Back Log"
+            ticketList={unassignedTickets}
+            companyId={company.id}
+            color="lightsalmon"
+          />
+        </div>
+        <div className="basis-1/4 text-center">
+          <TicketGroup
+            heading="In Progress"
+            ticketList={inProgressTickets}
+            companyId={company.id}
+            color="lightpink"
+          />
+        </div>
+        <div className="basis-1/4 text-center">
+          <TicketGroup
+            heading="Testing"
+            ticketList={testingTickets}
+            companyId={company.id}
+            color="skyblue"
+          />
+        </div>
+        <div className="basis-1/4 text-center">
+          <TicketGroup
+            heading="Verified"
+            ticketList={verifiedTickets}
+            companyId={company.id}
+            color="cadetblue"
+          />
+        </div>
+        <div className="basis-1/4 text-center">
+          <TicketGroup
+            heading="Approved"
+            ticketList={approvedTickets}
+            companyId={company.id}
+            color="yellowgreen"
+          />
+        </div>
+        {/* <div className="basis-1/4 text-center bg-yellow-400">
+          <h2 className="text-2xl underline  ">Unassinged</h2>
+          {unassignedTickets.map((ticket) => (
+            // <Ticket key={ticket.id} ticket={ticket}/>
+            <DragMove key={ticket.id} onDragMove={handleDragMove}>
+              <div
+                style={{
+                  transform: `translateX(${translate.x}px) translateY(${translate.y}px)`,
+                }}
+              >
+                <Ticket
+                  key={ticket.id}
+                  ticket={ticket}
```

Comment:
This file has a lot of white noise. I recommend removing any unnecessary code and using a linter to format your code.